### DISCUSSION TITLE
Training An Optimzed model for Mobile Deployment for Baby Crying etc

### DISCUSSION THREAD

**Discussion by m-ali-awan** [snakers4/silero-vad#242] [Q&A]
Hi there, first of all, I thank SILERO team for such wonderful work!

I want to develop a small, efficient audi classification model for laughing, crying etc. How can I finetune/train on the architecture of silero-vad, so the inference is done same as VAD is doing..?

Thanks, for any help

**Comment by snakers4** [comment]
Hi,

You can try using the VAD model as a feature extractor.

**Reply by m-ali-awan** [reply]
Hi there, thanks for the help.
Do we have any notebooks/resources, showing the way, how to use VAD as feature extractor for custom tasks?

Thanks

**Reply by dgoryeo** [reply]
Hi @m-ali-awan , did you find a solution? I'm interested in the same usecase and have not been able to find a solution yet. Let me know. Thanks.
